### ISSUE TITLE
When going to localhost3000, it says the extension isn't installed. (see pic)

### ISSUE THREAD

**Issue by simplyyyme** [rlesniak/tind3r.com#17]
https://i.gyazo.com/5c95bd18d5e32c9e40a63bf5de7f77ef.png

The server seems to be running fine.

**Comment by OrpheusXx** [comment]
I think i have the same problem. When i tried to install the new .crx fie it didn't work. Tried it in Vivaldi and Chrome too, but should be the same thing now.

**Comment by rlesniak** [comment]
Yeah, I know. I have recently fixed this issue but it needs newer version of extension which is taken down :( and updated version of this repo. I'll do it today (is online btw)

So you can update extension ID manually in https://github.com/rlesniak/tind3r.com/blob/master/src/const/index.js by editing `originalId` (this ID can be found under chrome://extensions/)

**Comment by rlesniak** [comment]
@simplyyyme @OrpheusXx try out now latest master

**Comment by OrpheusXx** [comment]
@rlesniak It seems to be working fine now, ty for the update!
I installed it by downloading the .crx file from the main page of tind3r, then dropped it into chrome://extensions/ in dev. mode. (using tind3r in Vivaldi actually, so it was vivaldi://extensions/ , but its the same)

Haven't tried building it the other way, i'm too nooby for that heh.
